Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNamesFakes.cs

```diff
@@ -12,44 +12,78 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Google.Protobuf.Collections;
+
 namespace Testing.ResourceNames
 {
-    public partial class SingleResource : ProtoMsgFake<SingleResource>
-    {
-        public string Name { get; set; }
-    }
-
-    public partial class SingleResourceRef : ProtoMsgFake<SingleResourceRef>
+    public partial class SinglePattern : ProtoMsgFake<SinglePattern>
     {
-        public string SingleResource { get; set; }
-        public string SingleResourceName { get; set; }
+        public string RealName { get; set; }
+        public string Ref { get; set; }
+        public RepeatedField<string> RepeatedRef { get; }
     }
 
-    public partial class MultiResource : ProtoMsgFake<MultiResource>
+    public partial class WildcardOnlyPattern : ProtoMsgFake<WildcardOnlyPattern>
     {
         public string Name { get; set; }
+        public string Ref { get; set; }
+        public RepeatedField<string> RepeatedRef { get; }
+        public string RefSugar { get; set; }
+        public RepeatedField<string> RepeatedRefSugar { get; }
     }
 
-    public partial class MultiResourceRef : ProtoMsgFake<MultiResourceRef>
-    {
-        public string MultiResource { get; set; }
-    }
-
-    public partial class FutureMultiResource : ProtoMsgFake<FutureMultiResource>
+    public partial class WildcardMultiPattern : ProtoMsgFake<WildcardMultiPattern>
     {
         public string Name { get; set; }
+        public string Ref { get; set; }
+        public RepeatedField<string> RepeatedRef { get; }
     }
 
-    public partial class OriginallySingleResource : ProtoMsgFake<OriginallySingleResource>
+    public partial class WildcardMultiPatternMultiple : ProtoMsgFake<WildcardMultiPatternMultiple>
     {
         public string Name { get; set; }
+        public string Ref { get; set; }
+        public RepeatedField<string> RepeatedRef { get; }
     }
 
-    public partial class OriginallySingleResourceRef : ProtoMsgFake<OriginallySingleResourceRef>
-    {
-        public string Resource1 { get; set; }
-        public string Resource2 { get; set; }
-    }
+    public partial class Response : ProtoMsgFake<Response> { }
+
+    //public partial class SingleResource : ProtoMsgFake<SingleResource>
```

Comment:
Can these be removed now? Not sure why they're here.

Reply:
Oops, yes, an oversight. Will fix.